frontend/src/static/css/config/index.scss

```
@@ -55,15 +55,15 @@ body {
 	--radius-48: 48px;
 	--radius-full: 999px;
 
-	--default-theme-color: #009933;
-	// --default-theme-color: #008ede;
+	--default-theme-color: var(--cinemata-green-600);
+	// --default-theme-color: var(--cinemata-strait-blue-400);
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)